### DISCUSSION TITLE
Support date variable field in post front matter

### DISCUSSION THREAD

**Discussion by onebree** [forem/forem#16389] [Feature Requests]
I have a set of old blog posts that I'd like to repost to my dev.to blog, but with publication dates in the past.

While the basic-markdown editor says Jekyll Front Matter is supported, I am unable to use the [date variable](https://jekyllrb.com/docs/front-matter/#predefined-variables-for-posts) for a custom publication date.

I do not want to portray my blog as being more active or recent than it actually is. I want to back-date posts as needed (in lieu of importing from third parties), and not have several appear with the same date.

**Comment by lightalloy** [comment]
A pr #16395 includes information on frontmatter date parsing and potential issues related to it.

**Comment by bixfrankonis** [comment]
I was looking into DEV but I'd literally need this before I could use it.
